packages/contracts/contracts/L1/ValidatorPool.sol

```diff
@@ -226,20 +261,57 @@ contract ValidatorPool is ReentrancyGuardUpgradeable, Semver {
     }
 
     /**
-     * @notice Increases the bond amount corresponding to the given output index.
+     * @notice Adds a pending bond to the challenge corresponding to the given output index and challenger address.
+     *         The pending bond is added to the bond when the challenge is completed,
+     *         and refunded when the challenge is canceled.
      *
-     * @param _challenger  Address of the challenger.
      * @param _outputIndex Index of the L2 checkpoint output.
+     * @param _challenger Address of the challenger.
      */
-    function increaseBond(address _challenger, uint256 _outputIndex) external {
+    function addPendingBond(uint256 _outputIndex, address _challenger) external onlyColosseum {
         Types.Bond storage bond = bonds[_outputIndex];
-        require(bond.expiresAt > 0, "ValidatorPool: the bond does not exist");
+        require(bond.expiresAt >= block.timestamp, "ValidatorPool: the bond does not exist");
```

Comment:
It should be like below. Btw, I think this requirement is unnecessary because we check creation period when create challenge.
```suggestion
        require(bond.expiresAt >= block.timestamp, "ValidatorPool: the output is already finalized");
```